gradlew

```diff
@@ -231,4 +231,16 @@ eval "set -- $(
         tr '\n' ' '
     )" '"$@"'
 
+# Check if the gradle task is a connector, if so, set INCLUDE_CONNECTORS so settings.gradle can
```

Comment:
This shouldn't be in `gradlew` since this file is "controlled" by the Gradle wrapper installation and upgrade process. When we run `./gradlew wrapper --gradle-version 7.3.3` or similar in the future to upgrade, we'd have to keep replacing this here.

Reply:
Ah, yeah I was unsure if that file was auto-generated or not. Okay, I'll try to think of a better place to put this.